index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<title>Fork Me! FCC: Test Suite Template</title>
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+</head>
+
+<body>
+	<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+	<header id="header">
+		<nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+			<div class="container-fluid">
+				<a class="navbar-brand" href="#" style="padding-left: 25px;">
+
+					<img id="header-img" class="d-inline-block align-text-top ml-5" height="24"
+						src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+						alt="original trombones logo">
+				</a>
+
+                <div class="d-flex">
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup"
+                        aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+				<div class="collapse navbar-collapse d-flex" id="navbarNavAltMarkup">                        
+                    <div class="navbar-nav float-lg-right">
+						<a class="nav-link" href="#features">Features</a>
+						<a class="nav-link" href="#how-it-works">How It Works</a>
+						<a class="nav-link" href="#pricing">Pricing</a>
+					</div>
+				</div>
+                </div>
+			</div>
+		</nav>
+	</header>
+
+	<div class="container" style="margin-top: 2rem;">
```

Comment:
- [ ] Please avoid CSS styling inside the HMTL file. move to the css/style.css instead.

Reply:
added `it keeps all styles centralized and readable.`